.github/workflows/cherryPick.yml

```
@@ -55,6 +58,20 @@ jobs:
           fi
           echo "VERSION_BUMP_SHA=$VERSION_BUMP_COMMIT" >> "$GITHUB_OUTPUT"
 
+      - name: Get Mobile-Expensify version bump commit
+        id: getMobileExpensifyVersionBumpCommit
+        working-directory: Mobile-Expensify
+        run: |
+          git switch main
+          VERSION_BUMP_COMMIT="$(git log --format='%H' --author='OSBotify' --grep 'Update version to ${{ needs.createNewVersion.outputs.NEW_VERSION }}')"
```

NAB - we can tell git to return directly after finding the first match
```suggestion
          VERSION_BUMP_COMMIT="$(git log -1 --format='%H' --author='OSBotify' --grep 'Update version to ${{ needs.createNewVersion.outputs.NEW_VERSION }}')"
```

it saves a nice amount of time locally for me

```
╰─❯ time git log --format='%H' --author='OSBotify' --grep 'Update version to 9.1.20-0'
94c07b6d9645d9c131c3b825a65e41b352194eac

________________________________________________________
Executed in    1.47 secs    fish           external
   usr time    1.41 secs    0.33 millis    1.40 secs
   sys time    0.06 secs    2.54 millis    0.06 secs

╰─❯ time git log -1 --format='%H' --author='OSBotify' --grep 'Update version to 9.1.20-1'
72d0bc226af135b914b63692482b6f0930b9bca3

________________________________________________________
Executed in   99.96 millis    fish           external
   usr time   30.18 millis    0.34 millis   29.83 millis
   sys time   34.27 millis    2.62 millis   31.64 millis
```

---

nice suggestion!